packages/flutter/test/widgets/editable_text_test.dart

```diff
@@ -107,6 +129,53 @@ void main() {
     expect(tester.testTextInput.setClientArgs!['inputAction'], equals(serializedActionName));
   }
 
+  // Related issue: https://github.com/flutter/flutter/issues/98115
+  testWidgets('test editable text scheduleShowCaretOnScreen with no animation when the window changes its metrics',
+          (WidgetTester tester)
+      async {
+
+    final ScrollController scrollController = ScrollController();
+    final Widget widget = MaterialApp(
+      home: Scaffold(
+        body: SingleChildScrollView(
+          controller: scrollController,
+          child: Column(
+            children: <Widget>[
+              Column(
+                children: List<Widget>.generate(
+                    5,
+                        (_) => Container(
+                      height: 1200.0,
+                      color: Colors.black12,
+                    )),
+              ),
+              SizedBox(
+                  height: 20,
+                  child: EditableText(
+                    controller: TextEditingController(),
+                    backgroundCursorColor: Colors.grey,
+                    focusNode: focusNode,
+                    style: const TextStyle(),
+                    cursorColor: Colors.red,
+                  )),
+            ],
+          ),
+        ),
+      ),
+    );
+
+    await tester.pumpWidget(
+        widget
+    );
+    await tester.showKeyboard(find.byType(EditableText));
+    TestWidgetsFlutterBinding.instance.window.viewInsetsTestValue = const _TestWindowPadding(bottom: 500);
+    await tester.pump();
+    final double offsetAfter = scrollController.offset;
+
+    // The offset of the scrollController should change immediately after window changes its metrics
+    expect(0,isNot(equals(offsetAfter)));
```

Comment:
space inbetween ,
```suggestion
    expect(0, isNot(equals(offsetAfter)));
```

Reply:
This seems reversed to me, maybe:

```dart
expect(offsetAfter, isNot(0.0));
```

I think the `equals` is optional here too.